iothub/service/src/Twin/Models/Twin.cs

```diff
@@ -66,28 +62,28 @@ public Twin(TwinProperties twinProperties)
         /// <summary>
         /// Gets and sets the twin tags.
         /// </summary>
-        public TwinCollection Tags { get; set; }
+        public TwinCollection Tags { get; set; } = new TwinCollection();
 
         /// <summary>
         /// Gets and sets the twin properties.
         /// </summary>
-        public TwinProperties Properties { get; set; }
+        public TwinProperties Properties { get; set; } = new TwinProperties();
 
         /// <summary>
         /// Gets the twin configuration properties.
         /// </summary>
         /// <remarks>
         /// Configuration properties are read only.
         /// </remarks>
-        public IDictionary<string, ConfigurationInfo> Configurations { get; internal set; }
+        public IDictionary<string, ConfigurationInfo> Configurations { get; internal set; } = new Dictionary<string, ConfigurationInfo>();
```

Comment:
```suggestion
        public IDictionary<string, ConfigurationInfo> Configurations { get; internal set; } = new();
```

Reply:
Can't because defined type is IDictionary.